actions/raw_action.go

```diff
@@ -1,3 +1,28 @@
+/*
+Raw Action
+
+Allows to write some file to output image with given offset.
+Usually this is needed for bootloader stuff.
```

Comment:
This is typically useful for bootloaders

Reply:
done